frontend/integration-tests/tests/olm/etcd.scenario.ts

```diff
@@ -51,8 +51,8 @@ describe('Interacting with the etcd OCS', () => {
     await browser.wait(until.textToBePresentInElement(crudView.rowForName(etcdOperatorName).$('a[title=pods]'), '1 of 1 pods'));
 
     expect(crudView.rowForName(etcdOperatorName).isDisplayed()).toBe(true);
-    expect(crudView.labelsForRow(etcdOperatorName).filter(l => l.getText().then(t => t === `alm-owner-name=${etcdOperatorName}`)).first()).toBeDefined();
-    expect(crudView.labelsForRow(etcdOperatorName).filter(l => l.getText().then(t => t === `alm-owner-namespace=${testName}`)).first()).toBeDefined();
+    expect(crudView.labelsForRow(etcdOperatorName).filter(l => l.getText().then(t => t === `olm.owner=${etcdOperatorName}`)).first()).toBeDefined();
+    expect(crudView.labelsForRow(etcdOperatorName).filter(l => l.getText().then(t => t === `olm.owner.namespace=${testName}`)).first()).toBeDefined();
```

Comment:
Can you test that putting a bogus value here fails the tests?

I suspect `toBeDefined()` is `true` even when the labels are missing.

Reply:
It looks like the label value we're checking is wrong for `olm.owner`. It should be `etcdoperator` instead of `etcd-operator`. The label checks aren't correctly failing when they should.